services/thermal_mgr/thermal_mgr.h

```diff
@@ -3,8 +3,11 @@
 #include "lm75bd.h"
 #include "errors.h"
 
+#define T_HYS 75
+
 typedef enum {
   THERMAL_MGR_EVENT_MEASURE_TEMP_CMD,
+  OS_MGR_EVENT_OVER_TEMP,
```

Comment:
This event should be called with THERMAL_MGR_EVENT_OS_OVER_TEMP instead as for this type of event we want to include the THERMAL_MGR_EVENT prefix